pkg/rpc/validators.go

```diff
@@ -19,17 +19,19 @@ type (
 		VotePubkey       string  `json:"votePubkey"`
 	}
 
+	VoteAccounts struct {
+		Current    []VoteAccount `json:"current"`
+		Delinquent []VoteAccount `json:"delinquent"`
+	}
+
 	GetVoteAccountsResponse struct {
-		Result struct {
-			Current    []VoteAccount `json:"current"`
-			Delinquent []VoteAccount `json:"delinquent"`
-		} `json:"result"`
-		Error rpcError `json:"error"`
+		Result VoteAccounts `json:"result"`
+		Error  rpcError1    `json:"error"`
 	}
```

Comment:
I changed it because I also changed the return type of `GetVoteAccounts` from `GetVoteAccountsResponse` to `VoteAccounts`, as it was only the `.Result` attribute (i.e., `VoteAccounts`) of the result that was ever used